Taskfile.yml

```diff
@@ -0,0 +1,32 @@
+version: '3'
+
+tasks:
+  default:
+    internal: false
+    silent: true
+    # yamllint disable rule:line-length
+    cmds:
+      - 'echo -e "\n👋 Welcome to Task! To see available commands, run: task -l"'
+      - 'echo -e "📝 To see instructions for a specific command run: task command-name --summary\n"'
+      - task -l --sort none
+    # yamllint enable rule:line-length
+
+  up:
+    desc: Start processes defined in `devenv.nix` in the background.
+    internal: false
+    silent: true
+    cmd: devenv up -d
+
+  down:
+    desc: Stop processes defined in `devenv.nix` in the background.
+    internal: false
+    silent: true
+    cmd: devenv processes down
+
+  test:
+    desc: Run pytest
+    internal: false
+    silent: true
+    cmds:
+      - pytest || true
+      - rm -rf ./tmp
```

Comment:
What's this file for?

Reply:
command shortcuts. It's like Makefile but uses Task

Reply:
devenv scripts and commands provide the same functionality without needing any extra dependency.

Reply:
Oh, wasn't aware of that. I can change it